data/json/mapgen/campus/buildings/media_f0.json

```diff
@@ -54,8 +54,7 @@
         "................................................",
         ".[....[....[...[..[...[...[..[........[...[.[..[",
         "......[...[.[..[...................[...[.[..[...",
-        "........................[...[.[..[..............",
-        "[..[.........[........[.................[........"
+        "........................[...[.[..[.............."
```

Comment:
Why not delete one of the identical rows above instead?

Reply:
Don't want to mess with the building outline because it needs to match the other floors.